classes/Translations/Translations.php

```diff
@@ -56,6 +56,13 @@ public function getTranslations()
             'general' => [
                 'save' => $this->module->l('Save', 'translations'),
                 'testModeOn' => $this->module->l('Test mode is turned on', 'translations'),
+                'multiShop' => [
+                    'title' => $this->module->l('Multistore detected', 'translations'),
+                    'subtitle' => $this->module->l('You must configure your shops one by one for this service, but you will be able to use the same accounts.', 'translations'),
```

Comment:
@LouiseBonnard, may we ask for your help on this PR?

```suggestion
                    'subtitle' => $this->module->l('Each shop must be configured separately, even if you configure the same account on all of them.', 'translations'),
```

Reply:
Looks clear to me!

Reply:
Done, thanks for help